### PR TITLE
Upload file by base64 and download file by get URL

### PR DESCRIPTION
**Pull request by malamili**
SDK-JS PR/Travis run: ...URL GOES HERE...

I certify that:
- [x] The automated build passes on the branch this pull request is from
- [x] My change is documented in the Changelist.md
- [x] Any new dockerimages this build depends on are properly tagged on github
- [x] I shall not merge until the travis build is green and the PR is accepted by
  another developer
- [x] I shall promptly fix/reply to comments, as I know PRs are not :fire: and
  forget
- [x] I shall keep my branch updated with any intermediate changes to the
  target branch
- [x] This is a quality PR
- [x] I got to the end of this list

### CODE REVIEW

File: src/file/FileCtrl.coffee [hunk 1]
```diff
@@ -103,6 +103,74 @@ server
...
+  buf = Buffer.from(req.body.file, 'base64')
+
+  FileService.uploadFile(req.query.fileName, req.query.target, buf)
+  .then((resol) ->
+    logger.code.debug(resol)
+    res.send(resol)
+  )
+  .catch((err) ->
+    logger.code.error(err)
+    res.status(500).send('Error somewhere')
```
Comment (Gielert) [inline src/file/FileCtrl.coffee]: I think this should be more descriptive

File: src/file/FileCtrl.coffee [hunk 2]
```diff
@@ -103,6 +103,74 @@ server
   )
 )
 
+
+# Download images as URL
+server
+.getApp()
+.get('/file', (req, res, next) ->
+  res.contentType('image/jpeg');
```
Comment (Gielert) [inline src/file/FileCtrl.coffee]: Do you know for sure it's alway an image and specifically a JPEG

Comment (Deam) [reply src/file/FileCtrl.coffee]: It is now saved as a PNG file, but it could be changed in the app if needed

Comment (malamili) [reply src/file/FileCtrl.coffee]: good point! will make this by a query variable

**Pull request merged**